Stresser/IEtwEventHandler.h

```diff
@@ -1,17 +1,17 @@
 #pragma once
-
-#include <Windows.h>
-
-// must be after Windows.h:
-#include <evntrace.h>
+#include "EtwEventTypes.h"
 
 /*
  * Represent class that have implemented ETW event callback for incoming events.
  */
 class IEtwEventHandler
 {
 public:
+	IEtwEventHandler(EtwEventTypes eventType);
+
 	virtual ~IEtwEventHandler() = default;
 
 	virtual void onEventRecord(PEVENT_RECORD record) = 0;
-};
\ No newline at end of file
+
+	EtwEventTypes eventType;
```

Comment:
Private member ?
Define `private` / `protected` and getter / setter functions if needed 🙌